### PR TITLE
Am tabbed interface

### PR DESCRIPTION
**Pull request by ajmompr**

### CODE REVIEW

File: app/views/users/liked.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Feed</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Followers</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Following</a>
+      </li>
```
Comment (JaredBears) [inline app/views/users/liked.html.erb:22]: You missed updated the links on the liked tab, but they work everywhere else.  Consider using a partial here instead that you can just reference on every page.

**Review by JaredBears — CHANGES_REQUESTED** [review]

File: app/views/users/liked.html.erb (hunk 1 above)
Comment (ajmompr) [reply app/views/users/liked.html.erb:22]: Hi Jared! Thank you for taking a look; I have updated my code to reflect your suggestions :)

**Review by JaredBears — APPROVED** [review]
It all works! :)

**Comment by CarlosCastanedaDev** [comment]
Code looks good!

**Pull request merged**